src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <a href="/" class="card__imgLink">
+        <img 
+          src="images/imac.jpeg" 
+          alt="iMac" 
+          class="card__img"
+        >
+      </a>
+
+      <a href="/" class="card__nameLink">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </a>
```

Comment:
use h2

Reply:
Thank you for your revision. All comments were corrected and I've also rechecked Checklist.